src/test/auxiliary/thread-local-extern-static.rs

```diff
@@ -0,0 +1,2 @@
+#[thread_local]
```

Comment:
These tests will need license header blocks, and I would *expect* at least that each test would require `#![feature(thread_local)]`, although I suspect the one below may not require it (could you make sure?)